modin/test/test_partition_api.py

```diff
@@ -83,6 +83,24 @@ def get_df(lib, data):
                     get_func(expected_partitions[row_idx][col_idx].list_of_blocks[0]),
                     get_func(actual_partitions[row_idx][col_idx]),
                 )
+        df = get_df(pd, data)
+        virtual_partitioned_df = pd.concat([df] * 10)
+        actual_partitions = np.array(
+            unwrap_partitions(virtual_partitioned_df, axis=axis)
+        )
+        assert expected_partitions.shape == actual_partitions.shape
+        expected_df = pd.concat([pd.DataFrame(get_df(pandas, data))] * 10)
```

Comment:
We don't need `get_df` in the new test case because we don't need to check for metadata propagation.